app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -541,6 +525,41 @@ class _CustomChatInputState extends ConsumerState<CustomChatInput> {
     return null;
   }
 
+  void attachmentConfirmation(
```

Comment:
So this is different from `attachmentConfirmationDialog`?

Reply:
This is meant only for chat attachment dialog, it has been revised on ben feedback that chat attachments should have its own functionality and not be mixed up with generic attachments flow. Later we can open ticket for suggestive refactor!